Makefile

```diff
@@ -78,7 +81,7 @@ test/unit: $(GINKGO)
 	INTERFACES_FILTER="" NODE_NAME=node01 $(GINKGO) $(unit_test_args) ./pkg/
 
 test/e2e: $(OPERATOR_SDK)
-	$(OPERATOR_SDK) test local ./test/e2e \
+	unset GOFLAGS && $(OPERATOR_SDK) test local ./test/e2e \
```

Comment:
What is the unset good for?

Reply:
operator-sdk is adding another mod=vendor so it fails.

Reply:
Please explain it in an inline comment for our future selves.